spec/unit/installer/xcode/multi_pods_project_generator_spec.rb

```diff
@@ -563,11 +563,11 @@ class Xcode
             target.stubs(:user_targets).returns([user_target])
 
             @generator = MultiPodsProjectGenerator.new(config.sandbox, [target], [],
-                                                       @analysis_result.all_user_build_configurations, @installation_options, config, '1')
+                                                       @analysis_result.all_user_build_configurations, @installation_options, config, 51)
             pod_generator_result = @generator.generate!
-            pod_generator_result.project.object_version.should == '1'
+            pod_generator_result.project.object_version.should == '51'
```

Comment:
this is the annoying part from Xcodeproj that returns this value back as a `String`.